### PR TITLE
https://github.com/stolostron/backlog/issues/26728

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/26728

### CODE REVIEW

**Review by birsanv — APPROVED** [review]
lgtm

**Comment by xuezhaojun** [comment]
@oafischer Thanks for the input! The content looks good to me!
A little issue is I think we missed the https://github.com/stolostron/backlog/issues/26728#issuecomment-1308321646, and .DS_Store file is mis-uploaded.

**Comment by oafischer** [comment]
Thanks for pointing that out @xuezhaojun! Added fix from comment. The .DS_Store file can't be changed. Ready for final review.

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -561,6 +561,17 @@ do curl -k oc get infraenv qe2 -ojsonpath="{.status.bootArtifacts.${artifact}}"`
 
 . Add the `ipxeScript` artifact proxy URL to the `bootp` parameter in `libvirt.xml`.
 
+[#cannot-delete-clusterdeployment]
+=== Cannot delete ClusterDeployment after upgrading {product-title-short}
+
+If you are using the removed BareMetalAssets API in {product-title-short} 2.6, the ClusterDeployment cannot be deleted after upgrading to {product-title-short} 2.7 because the BareMetalAssets API is bound to the ClusterDeployment.
```
Comment (cadawson) [inline release_notes/known_issues.adoc]: Should "ClusterDeployment" be in monospace?

**Review by cadawson — APPROVED** [review]
If this is covered in the deprecated/removed topic, it looks good. Thanks!

File: release_notes/known_issues.adoc (hunk 1 above)
Comment (oafischer) [reply release_notes/known_issues.adoc]: It's a `kind` so probably yes, thanks for catching that!

**Pull request merged**